libc/functions/c/clearenv.part-impl.md

```diff
@@ -0,0 +1,35 @@
+# Synopsis 
+`#include <stdlib.h>`</br>
+
+`int clearenv(void);`</br>
+
+## Status
+Partially implemented
+## Conformance
+Various UNIX variants (DG/UX, HP-UX, QNX, ...)
+## Description
+
+
+The clearenv() function clears the environment of all name-value pairs and sets the value of the external variable environ to NULL. After this call, new variables can be added to the environment using putenv() and setenv().
```

Comment:
Please place function names, numbers, errnos, and other non-words in ``
Applies to the rest of code within this PR.

```suggestion
The `clearenv()` function clears the environment of all name-value pairs and sets the value of the external variable environ to `NULL`. After this call, new variables can be added to the environment using `putenv()` and `setenv()`.
```